tables-book/04-01-demographics.Rmd

```diff
@@ -233,6 +233,72 @@ ft <- summarizor(adsl, by = "ARM") |>
 ft
 ```
 
+
+### tfrmt
+
+Please note that the`tfrmt` package is intended for use with mock data or ARD (analysis results data). 
+This package creates similar tables to other packages but requires the dataset to be in a specific format. 
+In this bookdown `data_demog` and `data_ae` are used within `tfrmt` examples (these datasets are stored in this package too). 
+These are based on the CDISC pilot data but have been modified into ARD - hence, differ from the other packages examples. 
+
+To see how these datasets have been created please refer to the following: 
+<ul>
+<li> https://github.com/GSK-Biostatistics/tfrmt/blob/main/data-raw/create_data_demog.R </li>
+<li> https://github.com/GSK-Biostatistics/tfrmt/blob/main/data-raw/create_data_ae.R </li>
+</ul>
+
+```{r}
+resetSession()
+library(tfrmt)
+library(dplyr)
+
+# Get data
+data_demog <- tfrmt::data_demog
```

Comment:
hi @karima662,

You again seem to be using different starting data than all other code sections in the package.

A core goal of the book is to provide specific, comparable usages across multiple table engines.

This means that we need you to actually use the same data as everyone else. And, again, we need the code to transform that raw CDISC data into your ARD to be in the book, not simply blind linked to a separate location.

Can you please rewrite the example using the data we have selected?

Reply:
- Please can you review this PR: https://github.com/RConsortium/rtrs-wg/pull/108
- I have re-written the demographics table example starting from the CDISC data